Example/RealmResultsController-iOSTests/RealmLoggerTests.swift

```diff
@@ -22,6 +22,16 @@ class NotificationListener {
     }
 }
 
+class NotificationObserver {
+    
+    var notificationReceived: Bool = false
+    init() {
+        NSNotificationCenter.defaultCenter().addObserverForName("Task-123", object: nil, queue: nil) { (notification) -> Void in
```

Comment:
yes, autocomplete, this is a Dummy class for testing, i think doesn't matter much...